modules/speller/default/suggest.cpp

```diff
@@ -74,7 +77,10 @@ using namespace std;
 
 namespace {
 
-  typedef vector<String> NearMissesFinal;
+  // The internal, C++ version of the external Suggest/AspellSuggest
+  // struct.  I don't use that struct because I want the convenience
+  // of automatically allocated space for the strings.
+  typedef vector<pair<String,int> > NearMissesFinal;
```

Comment:
comment from https://github.com/GNUAspell/aspell/issues/365#issuecomment-266261921
```
A "String" as a struct member will behave in the exact
same fashion of a "String" in a pair. Please use a struct to allow
for the inclusion of additional information in the future.
```
but I'm not clear on the scope of this change.